nixos/modules/services/web-servers/garage.nix

```diff
@@ -71,6 +64,19 @@ in
   };
 
```

Comment:
Try using `lib.mkRemovedOptionModule` instead, though I'm not sure if that works with submodules (`mkRenamedOptionModule` doesn't, see #96006)

```nix
{
  imports = [
    (lib.removedOptionModule ["services" "garage" "settings" "replication_mode"] ''
      Garage requires an explicit replication factor to be set since version 1.0.0.
        
      Please set `services.garage.settings.replication_factor` to 1 explicitly to preserve the previous behavior.
      See https://git.deuxfleurs.fr/Deuxfleurs/garage/src/tag/v1.0.0/doc/book/reference-manual/configuration.md#replication_factor for more info.
    '')
  ];
    
  config = mkIf cfg.enable {
    # ...
  }
}
```